### DISCUSSION TITLE
Loading indicator on widgets

### DISCUSSION THREAD

**Discussion by oliverbj** [filamentphp/filament#13764] [Help]
### Package

Widgets

### Package Version

v3.2.96

### How can we help you?

Hi.

I was wondering if its possible to show a loading indicator (e.g., a skeleton loader) on widgets that takes some time to load?

I have a widget which is getting data from a MSSQL connection - and it takes some time to load it. Currently, the widget is just "blank" until the data has been loaded.

**Comment by leandrocfe** [comment]
Custom widget?

```html
<div wire:loading> 
        <x-filament::loading-indicator class="h-5 w-5" />
</div>
```
